paper-api/src/main/java/org/bukkit/inventory/ItemStack.java

```diff
@@ -64,10 +64,26 @@ public class ItemStack implements Cloneable, ConfigurationSerializable, Translat
     // Paper end
 
     // Paper start - pdc
+    /**
+     * @see #editPersistentDataContainer(Consumer)
+     */
```

Comment:
Should this mention that the view is readonly?

Reply:
Yea I guess the method would benefit from proper javadocs. If you can push it, I am not around rn.

Reply:
The javadocs in the PersistentDataViewHolder interface which this method is from already seems good to me, it states that it's only capable of viewing tags, not modifying them.